config/initializers/redis.rb

```diff
@@ -0,0 +1,6 @@
+# DEV uses the RedisCloud Heroku Add-On which comes with the predefiend env variable REDISCLOUD_URL
+redis_url = ENV["REDISCLOUD_URL"]
+redis_url ||= ApplicationConfig["REDIS_URL"]
+
+uri = URI.parse(redis_url)
+RedisClient = Redis.new(host: uri.host, port: uri.port)
```

Comment:
I didn't catch it the first time around but... shouldn't we pass the entire URL to the Redis constructor? Otherwise we'd lose the ability to have users use multiple DBs or any other possible param in the custom URL.

Got this from the redis gem readme - https://github.com/redis/redis-rb#getting-started:

```ruby
redis = Redis.new(url: "redis://:p4ssw0rd@10.0.1.1:6380/15")
```